docs/deploy/self-hosted/high-availability/configuration.mdx

```diff
@@ -4,6 +4,15 @@ title: Configuration
 
 ## Synchronous Replication
 
+Between physical replicas, ParadeDB requires the use of a few settings (which are automatically set by [CNPG](/deploy/self-hosted/kubernetes)) in order to avoid query cancellation due to ongoing reorganization of the data on the primary replica.
+
+- `hot_standby_feedback=on` - The [`hot_standby_feedback`](https://www.postgresql.org/docs/current/runtime-config-replication.html#GUC-HOT-STANDBY-FEEDBACK) setting controls whether nodes acting as `hot_standby`s (the replicas in physical replication) send feedback to the leader about their current transaction status. ParadeDB uses this transaction status to determine when it is safe for the primary to garbage collect its segments.
+- `primary_slot_name=$something` - The [`primary_slot_name`](https://www.postgresql.org/docs/current/runtime-config-replication.html#GUC-PRIMARY-SLOT-NAME) setting declares the name of the replication slot that a replica should use when it connects to the primary. In order for `hot_standby_feedback` to be used and persistent, a replication slot must be used.
```

Comment:
question:  if a cluster has multiple hot standby nodes, does each one need a unique name, or does it not matter?

Reply:
Each client must use a separate slot, AFAIK.

This is already handled by CNPG, because using slots is best practice... I'm less concerned that folks won't be using slots than the other setting.